_lib/core/common.php

```diff
@@ -543,7 +544,7 @@ function send_mail($opts = [])
 {
     require 'vendor/autoload.php'; // If you're using Composer (recommended)
     
-    $email = new \SendGrid\Mail\Mail();
+    $email = new Mail();
```

Comment:
will this still work?

Reply:
It should be fine as we have the import at the top of the page.

Reply:
@adamjimenez Where does that class actually live?

Reply:
Should it be a composer dependancy?